src/main/kotlin/test/ktortemplate/core/utils/json/JsonSettings.kt

```diff
@@ -0,0 +1,23 @@
+package test.ktortemplate.core.utils.json
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.google.gson.reflect.TypeToken
+import test.ktortemplate.core.model.Car
+import test.ktortemplate.core.utils.pagination.PageResponse
+import java.lang.reflect.Type
+
+object JsonSettings {
+
+    val PageResponseCarType: Type = object : TypeToken<PageResponse<Car>>() {}.type
```

Comment:
The typed PageResponse \<Car\> must be added here, and registered below on the gson mapper